configure.go

```diff
@@ -0,0 +1,118 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/codegangsta/cli"
+	"github.com/jrperritt/rack/util"
+	"gopkg.in/ini.v1"
+)
+
+var availableRegions = []string{"DFW", "IAD", "ORD", "LON", "SYD", "HKG"}
+
+func configure(c *cli.Context) {
+	intro := []string{"\nThis interactive session will walk you through creating",
+		"a profile in your configuration file. You may fill in all or none of the",
+		"values.\n"}
+	fmt.Println(strings.Join(intro, "\n"))
+	reader := bufio.NewReader(os.Stdin)
+	m := map[string]string{
+		"username": "",
+		"apikey":   "",
+		"region":   "",
+	}
+	fmt.Print("Rackspace Username: ")
+	username, _ := reader.ReadString('\n')
+	m["username"] = strings.TrimSuffix(username, string('\n'))
+
+	fmt.Print("Rackspace API key: ")
+	apiKey, _ := reader.ReadString('\n')
+	m["apikey"] = strings.TrimSuffix(apiKey, string('\n'))
+
+	regionPrompt(m, reader)
+
+	fmt.Print("Profile Name: ")
+	profile, _ := reader.ReadString('\n')
+	profile = strings.TrimSuffix(profile, string('\n'))
+
+	configFile, err := configFile()
+	var cfg *ini.File
+	cfg, err = ini.Load(configFile)
+	if err != nil {
+		fmt.Printf("Error loading config file: %s\n", err)
+		cfg = ini.Empty()
+	}
+
+	if profile == "" {
+		profile = "DEFAULT"
+	}
+
+	checkIfProfileExists(cfg, profile, reader)
+
+	section, err := cfg.NewSection(profile)
+	if err != nil {
+		fmt.Printf("Error creating new section [%s] in config file: %s\n", profile, err)
+		return
+	}
+
+	for key, val := range m {
+		section.NewKey(key, val)
+	}
+
+	err = cfg.SaveTo(configFile)
+	if err != nil {
+		fmt.Printf("Error saving config file: %s\n", err)
+		return
+	}
+}
+
+func regionPrompt(m map[string]string, reader *bufio.Reader) {
+	fmt.Printf("Rackspace Region (%s): ", strings.Join(availableRegions, ", "))
+	region, _ := reader.ReadString('\n')
+	region = strings.ToUpper(strings.TrimSuffix(region, string('\n')))
+	if util.Contains(availableRegions, region) {
```

Comment:
This would be nice to have. Only downside would be that if Rackspace adds a region, users would have to download the CLI again to use that new region.